### ISSUE TITLE
fasterq-dump freezes at "verifying CA cert" stage

### ISSUE THREAD

**Issue by austinreidmanny** [ncbi/sra-tools#154]
I'm trying to use fasterq-dump to retrieve a transcriptome from the SRA, but it freezes every time at the same step. I ran it in verbose mode and have attached the output here.

```
$ fasterq-dump SRX2311572 --split-3 -t /tmp/ -v --outdir ./ -e 4
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to www.ncbi.nlm.nih.gov (130.14.29.110) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to sra-download.ncbi.nlm.nih.gov (130.14.250.24) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to www.ncbi.nlm.nih.gov (130.14.29.110) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to sra-download.ncbi.nlm.nih.gov (130.14.250.24) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to www.ncbi.nlm.nih.gov (130.14.29.110) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to sra-download.ncbi.nlm.nih.gov (130.14.250.24) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to www.ncbi.nlm.nih.gov (130.14.29.110) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to www.ncbi.nlm.nih.gov (130.14.29.110) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to www.ncbi.nlm.nih.gov (130.14.29.110) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to www.ncbi.nlm.nih.gov (130.14.29.110) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to sra-download.ncbi.nlm.nih.gov (130.14.250.24) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to sra-download.ncbi.nlm.nih.gov (130.14.250.24) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to sra-download.ncbi.nlm.nih.gov (130.14.250.24) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - connected from '10.119.225.147' to sra-download.ncbi.nlm.nih.gov (130.14.250.24) 
2018-09-21T12:55:02 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:03 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:03 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
2018-09-21T12:55:03 fasterq-dump.2.9.2: KClientHttpOpen - verifying CA cert 
```

**Comment by kwrodarmer** [comment]
Please re-run with the run accession `SRR4734700` rather than the experiment accession you gave, then send us the results. You may also want to run `test-sra SRR4734700` and capture the results to a file, which you should send to sra-tools@ncbi.nlm.nih.gov .

Thank you

**Comment by austinreidmanny** [comment]
Hello,

The problem is resolved. The issue was two-fold: 

1) I only had a single -v flag, which wasn’t that verbose. The run would quietly fail with the last message related to network I/O, so I thought it was that. 3 -v flags revealed it was connecting fine. 

 2) Cache directory: I had set the --outdir as the current working directory, and I set the tmp directory to scratch space, so it wouldn’t fill up my home directory. But I did not realize that there was a Cache directory which was set by default to a folder in my home directory. That was filling up really quick and causing it to fail. I changed that path to scratch space and it works fine.

Best,
Austin
